DonjinKrawler-Client/src/main/java/donjinkrawler/EnemyShell.java

```diff
@@ -0,0 +1,50 @@
+package donjinkrawler;
+
+import javax.swing.*;
+import java.util.Random;
+
+public class EnemyShell extends AbstractShell {
+
+    public EnemyShell(String name) {
+        this.name = name;
+        Random random = new Random();
+        this.x = random.nextInt(250);
+        this.y = random.nextInt(250);
+        loadImage();
+    }
+
+    protected void loadImage() {
+
+        ImageIcon ii = new ImageIcon(ClassLoader.getSystemResource(this.resolveImageName()).getFile());
+        image = ii.getImage();
+    }
+
+    private String resolveImageName()
+    {
+        if(this.getName().equals("Small Zombie")) {
+            return "zombie.png";
```

Comment:
We could use a Map here probably, to spare the ugly ifs since we can't use switch statements.

Reply:
As i can see you are realy hurt about those switch statmens :D

Reply:
Sito nematau reikalo taisyti kol kas nes kam kurti nauja hash map objecta, cia ofc tiktu tikrai labai tie nauji switch statmentai ;)

Reply:
Na vietoj visų ifų tau tereiktų panaudoti Map'o get metodą, ir kai norėsi naujų mobų pridėt vietoj to, kad pūsti šį metodą, užteks pridėti vieną eilutę prie immutable map. Žinant tavo rašymo gebėjimus, pakūrus naują mobą gali ne tokį magic stringą įrašyti ir tada baba, neturim mob'o. But okay, I'll fold this time